app/views/catalog/_show_main_content.html.erb

```diff
@@ -17,7 +17,7 @@
 
   </div>
 
-  <%= render partial: 'show_availability', locals: { document: @document } if @document && !blackcat_config.blackcat_message?(:readonly) %>
+  <%= render partial: 'show_availability', locals: { document: @document } if @document unless ENV['readonly'] %>
```

Comment:
`if @document unless ENV['readonly']` would this work 🤔

Reply:
It did work in my testing, were you able to test?

Reply:
it worked yup ✔️